tests/scale/test_load.py

```diff
@@ -0,0 +1,41 @@
+import logging
+import pytest
+import uuid
+
+import config
+import jenkins
+import time
+
+@pytest.mark.scale
```

Comment:
Please fix spacing.  2 newlines between methods.  No need for so many newlines at the end.